### PR TITLE
feature/impl

### PR DESCRIPTION
**Pull request by cichan02**

### CODE REVIEW

File: src/main/java/by/piskunou/solvdlaba/domain/Airport.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package by.piskunou.solvdlaba.domain;
+
+import lombok.*;
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/domain/Airport.java:3]: don't use *

File: src/main/java/by/piskunou/solvdlaba/domain/Airport.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "airports")
+public class Airport {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/domain/Airport.java:13]: hashcode is not overridden

File: src/main/java/by/piskunou/solvdlaba/persistence/impl/AirportRepositoryImpl.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+import org.springframework.r2dbc.core.DatabaseClient;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Repository
+@RequiredArgsConstructor
+public class AirportRepositoryImpl implements AirportRepository {
+
+    private final R2dbcAirportRepository r2dbcRepository;
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/persistence/impl/AirportRepositoryImpl.java:17]: r2dbc prefix is redundant

File: src/main/java/by/piskunou/solvdlaba/service/impl/CountryServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    @Override
+    public Mono<Boolean> existsById(long id) {
+        return repository.isExistsById(id);
+    }
+
+    @Override
+    public Mono<Boolean> isExistsByName(String name, Long id) {
+        long legalId = id == null ? 0 : id;
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/service/impl/CountryServiceImpl.java:64]: better to use 
```java
if(){
} else{
}
```

File: src/main/resources/application.yaml [hunk 5]
```diff
@@ -2,6 +2,21 @@ spring:
...
+    username:
+    password:
+
+  #Liquibase
+  liquibase:
+    enabled: true
+    change-log: classpath:db.changelog/db.changelog-master.yaml
+    driver-class-name: org.postgresql.Driver
+    url:
+    user: ${spring.r2dbc.username}
```
Comment (h1alexbel) [inline src/main/resources/application.yaml:17]: don't depend on spring.r2dbc.username and password

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml [hunk 6]
```diff
@@ -14,8 +14,6 @@ spring:
     change-log: classpath:db.changelog/db.changelog-master.yaml
     driver-class-name: org.postgresql.Driver
     url:
-    user: ${spring.r2dbc.username}
-    password: ${spring.r2dbc.password}
```
Comment (h1alexbel) [inline src/main/resources/application.yaml]: Liquibase needs `user` and `password`. Specify them, but using env

File: src/main/resources/application.yaml [hunk 7]
```diff
@@ -14,6 +14,8 @@ spring:
     change-log: classpath:db.changelog/db.changelog-master.yaml
     driver-class-name: org.postgresql.Driver
     url:
+    user:
```
Comment (h1alexbel) [inline src/main/resources/application.yaml:17]: @cichan02 
empty values? or I miss something..

**Comment by cichan02** [comment]
@h1alexbel May I merge this PR? Or are you still have been reviewing?

**Comment by cichan02** [comment]
@h1alexbel May You (or I) merge this PR? Or is the review still in progress?

**Pull request merged**